pkgs/development/python-modules/pint/default.nix

```
@@ -1,21 +1,26 @@
-{ stdenv
+{ lib
 , buildPythonPackage
 , fetchPypi
+, isPy27
+, funcsigs
 }:
 
 buildPythonPackage rec {
-  pname = "pint";
+  pname = "Pint";
```

```suggestion
  pname = "pint";
```
Put the line inside `fetchPypi` instead.

---

fixed.